src/config/typeConfigs.ts

```diff
@@ -210,6 +211,14 @@ export const typeConfigList: Type[] = [
     visibility: [FlowTypes.MESSAGING_OFFLINE, FlowTypes.VOICE]
   },
 
+  {
+    type: Types.wait_for_dial,
+    name: i18n.t('actions.wait_for_dial.name', 'Wait for Call'),
+    description: i18n.t('actions.wait_for_dial.description', 'Wait for a call to another number'),
+    form: DialRouterForm,
+    visibility: VISIBILITY_VOICE
```

Comment:
@ericnewcomer was thinking to have a component which renders the phone number.. but it seems like components are setup for actions and the other splits don't render anything, so not sure..

Reply:
As in a temba-component? This seems like a useful thing to include over there (which we could then use directly here) `<temba-phone/>`

Reply:
No the component which renders content on the node (e.g. `SendMsgComp`). But we don't do this for other split types (e.g. you can't see the expression on a Split By Expression node) so I think it's ok without it.

Reply:
Ah, well, in any case, seems useful enough to have it live in temba-components so we could use it everywhere.